### PR TITLE
Add Volume Scroll plugin

### LINKED ISSUE #51: Volume Scroll
I'm missing the ability to scroll the volume icon/bar to change the volume, could you add this as a little plugin?

I thought of making my own plugin, but currently making plugins seems like a lot of boilerplate for such a simple modification

### PR DESCRIPTION
**Pull request by n1ckoates**
I didn't want to setup my own plugin repo for just 1 plugin, and yours has TypeScript support and everything, so I hope it's fine if I just add this plugin here.

Closes #51

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,10 @@ Stream size            : 525 MiB (100%)
 MD5 of the unencoded content: 1401A5C520F16BADDEA6670EFF08200B
 ```
 
+## NoBuffer
```
Comment (Inrixia) [inline README.md:116]: This should be Volume Scroll or whatever you want the title to be :)

File: plugins/VolumeScroll/package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Inrixia) [inline plugins/VolumeScroll/package.json:1]: You don't need to require my lib if you aren't using anything from it.

**Review by Inrixia — CHANGES_REQUESTED** [review]

**Comment by Inrixia** [comment]
Absolutely no issue :) happy to merge this once stuff is sorted.

**Comment by Araxeus** [comment]
Lol I actually made one too https://github.com/Araxeus/tidal-neptune-plugins

File: README.md (hunk 1 above)
Comment (n1ckoates) [reply README.md]: Good catch, forgot to change it when copy/pasting 😅

**Comment by Inrixia** [comment]
Oh also on a unrelated note you should put the pkugin above the Song Downloader one in the Readme as I always keep that at the bottom of the list.

**Comment by n1ckoates** [comment]
sorted all of those issues 👍

**Comment by Inrixia** [comment]
Great thanks!

**Pull request merged**